### PR TITLE
Fix memory options for some functions

### PR DESCRIPTION
**Pull request by inlined**
For context, see alpha tester report

### CODE REVIEW

**Review by taeold — APPROVED** [review]
one minor comment, lgtm.

File: src/gcp/cloudfunctionsv2.ts [hunk 1]
```diff
@@ -163,6 +163,40 @@ interface GenerateUploadUrlResponse {
   storageSource: StorageSource;
 }
 
+// AvailableMemory suffixes and their byte count.
+type MemoryUnit = "" | "k" | "M" | "G" | "Mi" | "Gi";
```
[the remark this answers is not in the document]
Comment (colerogers) [reply src/gcp/cloudfunctionsv2.ts:167]: From the GCF proto, it looks like `Ki` is not a supported unit.

**Review by colerogers — APPROVED** [review]
LGTM

File: src/gcp/cloudfunctionsv2.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (inlined) [reply src/gcp/cloudfunctionsv2.ts:167]: I can just extend this to support more than GCF supports pretty easily.

**Pull request merged**